include/ofi_mr.h

```diff
@@ -124,6 +124,7 @@ struct ofi_mr_cache;
 
 union ofi_mr_hmem_info {
 	uint64_t cuda_id;
+	uint64_t mem_id;
```

Comment:
Please rename to ze_mem_id or similar to clarify which API this field is associated with.

Reply:
Sure.